R/pkg/R/mllib.R

```diff
@@ -1828,13 +1849,13 @@ setMethod("summary", signature(object = "RandomForestRegressionModel"),
 #' @note summary(RandomForestClassificationModel) since 2.1.0
 setMethod("summary", signature(object = "RandomForestClassificationModel"),
           function(object) {
-            ans <- summary.randomForest(object)
+            ans <- summary.treeEnsemble(object)
             class(ans) <- "summary.RandomForestClassificationModel"
             ans
           })
 
 #  Prints the summary of Random Forest Regression Model
-print.summary.randomForest <- function(x) {
+print.summary.treeEnsemble <- function(x) {
```

Comment:
That's OK. I thought the output may be very large which will flood the screen. The output of ```toDebugString``` is also not very legible compared with ```rpart```. I like the idea to make summary string consistent between languages. Let's get this in firstly and improve ```toDebugString``` at Scala side in a separate task which can also benefit SparkR.